app/services/billable_metrics/update_service.rb

```diff
@@ -24,7 +24,7 @@ def call
         if params.key?(:filters)
           BillableMetricFilters::CreateOrUpdateBatchService.call(
             billable_metric:,
-            filters_params: params[:filters].map(&:with_indifferent_access),
+            filters_params: params[:filters].map { |f| f.to_h.with_indifferent_access }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)